src/olympia/files/tests/test_views.py

```diff
@@ -24,12 +24,19 @@
 from olympia.users.models import UserProfile
 
 
-dictionary = 'src/olympia/files/fixtures/files/dictionary-test.xpi'
+files_fixtures = 'src/olympia/files/fixtures/files/'
 unicode_filenames = 'src/olympia/files/fixtures/files/unicode-filenames.xpi'
 not_binary = 'install.js'
 binary = 'dictionaries/ar.dic'
 
 
+def create_directory(path):
```

Comment:
if we use default_storage and the associated methods we can don't need to worry about directories existing (?)

Reply:
I think we still do, we do all over the place. But I didn't check in-depth but mostly tried to leave the test as it was.

Reply:
we do because its old code and we forget to use storage :)